frontend/public/components/graphs/index.tsx

```diff
@@ -7,6 +7,7 @@ import { ONE_HOUR } from '@console/shared/src/constants/time';
 export const PROMETHEUS_BASE_PATH = window.SERVER_FLAGS.prometheusBaseURL;
 export const PROMETHEUS_TENANCY_BASE_PATH = window.SERVER_FLAGS.prometheusTenancyBaseURL;
 export const ALERTMANAGER_BASE_PATH = window.SERVER_FLAGS.alertManagerBaseURL;
+export const ALERTMANAGER_USER_WORKLOAD_BASE_PATH = window.SERVER_FLAGS.alertManagerUserWorkloadBaseURL;
```

Comment:
```suggestion
export const ALERTMANAGER_USER_WORKLOAD_BASE_PATH =
  window.SERVER_FLAGS.alertManagerUserWorkloadBaseURL;
```

This should fix the CI lint test error.